tests/storage_test.go

```diff
@@ -71,15 +72,20 @@ var _ = Describe("Storage", func() {
 		var vmi *v1.VirtualMachineInstance
 		var nfsInitialized bool
 
-		initNFS := func() string {
+		initNFS := func(isIpv6 bool) string {
 			if !nfsInitialized {
 				_pvName = "test-nfs" + rand.String(48)
 				// Prepare a NFS backed PV
 				By("Starting an NFS POD")
 				os := string(cd.ContainerDiskAlpine)
-				nfsIP := tests.CreateNFSTargetPOD(os)
+				nfsIPs := tests.CreateNFSTargetPOD(os)
 				// create a new PV and PVC (PVs can't be reused)
 				By("create a new NFS PV and PVC")
+				nfsIP := libnet.GetIp(libnet.GetPodIpsStrings(nfsIPs), isIpv6)
+				if nfsIP == "" && isIpv6 {
+					Skip("skipping ipv6 nfs test on a single stack cluster")
```

Comment:
You could skip earlier, couldn't you ? 

Since the ipv6 / dual stack is a cluster-wide config, why don't you access a well-known pod on the system, and check its IPs ?

Reply:
well... yes. Done.